src/user-event/type/type.ts

```diff
@@ -79,14 +78,16 @@ export async function type(
   dispatchEvent(element, 'blur', EventBuilder.Common.blur());
 }
 
-async function emitTypingEvents(
+export async function emitTypingEvents(
+  config: UserEventConfig,
```

Comment:
[nitpick] i often see config options passed as the last argument (usually because they're optional), which makes me wonder if we always want these options? could we want a default config ?

Reply:
This is an internal function (export is not added to `user-event/index.ts`), we need the current config in order to invoke `wait`, etc